ipynb/AG_stool_temporal dynamics.ipynb

```diff
@@ -0,0 +1,664 @@
+    "            uni_sub_del = uni_sub_del.append(absdeltas.loc[dl], ignore_index=True)\n",
+    "\n",
+    "            \n",
+    "    elif absdeltas['DeltaInMonths'][dl] == 6:\n",
+    "        if (absdeltas['HOST_SUBJECT_ID'][dl] not in host_6mo):\n",
+    "            host_6mo.update({absdeltas['HOST_SUBJECT_ID'][dl]:absdeltas['HOST_SUBJECT_ID'][dl]})\n",
+    "            uni_sub_del = uni_sub_del.append(absdeltas.loc[dl], ignore_index=True)\n",
+    "    \n",
+    "    elif absdeltas['DeltaInMonths'][dl] == 7:\n",
+    "        if (absdeltas['HOST_SUBJECT_ID'][dl] not in host_7mo):\n",
+    "            host_7mo.update({absdeltas['HOST_SUBJECT_ID'][dl]:absdeltas['HOST_SUBJECT_ID'][dl]})\n",
+    "            uni_sub_del = uni_sub_del.append(absdeltas.loc[dl], ignore_index=True)\n",
+    "\n",
+    "    elif absdeltas['DeltaInMonths'][dl] == 8:\n",
+    "        if (absdeltas['HOST_SUBJECT_ID'][dl] not in host_8mo):\n",
+    "            host_8mo.update({absdeltas['HOST_SUBJECT_ID'][dl]:absdeltas['HOST_SUBJECT_ID'][dl]})\n",
+    "            uni_sub_del = uni_sub_del.append(absdeltas.loc[dl], ignore_index=True)\n",
+    "    \n",
+    "    elif absdeltas['DeltaInMonths'][dl] == 9:\n",
+    "        if (absdeltas['HOST_SUBJECT_ID'][dl] not in host_9mo):\n",
+    "            host_9mo.update({absdeltas['HOST_SUBJECT_ID'][dl]:absdeltas['HOST_SUBJECT_ID'][dl]})\n",
+    "            uni_sub_del = uni_sub_del.append(absdeltas.loc[dl], ignore_index=True)\n",
+    "            \n",
+    "    elif absdeltas['DeltaInMonths'][dl] == 10:\n",
+    "        if (absdeltas['HOST_SUBJECT_ID'][dl] not in host_10mo):\n",
+    "            host_10mo.update({absdeltas['HOST_SUBJECT_ID'][dl]:absdeltas['HOST_SUBJECT_ID'][dl]})\n",
+    "            uni_sub_del = uni_sub_del.append(absdeltas.loc[dl], ignore_index=True)\n",
+    "            \n",
+    "    elif absdeltas['DeltaInMonths'][dl] == 11:\n",
+    "        if (absdeltas['HOST_SUBJECT_ID'][dl] not in host_11mo):\n",
+    "            host_11mo.update({absdeltas['HOST_SUBJECT_ID'][dl]:absdeltas['HOST_SUBJECT_ID'][dl]})\n",
+    "            uni_sub_del = uni_sub_del.append(absdeltas.loc[dl], ignore_index=True)\n",
+    "            \n",
+    "    elif absdeltas['DeltaInMonths'][dl] == 12:\n",
+    "        if (absdeltas['HOST_SUBJECT_ID'][dl] not in host_12mo):\n",
+    "            host_12mo.update({absdeltas['HOST_SUBJECT_ID'][dl]:absdeltas['HOST_SUBJECT_ID'][dl]})\n",
+    "            uni_sub_del = uni_sub_del.append(absdeltas.loc[dl], ignore_index=True)\n",
+    "            \n",
+    "    elif absdeltas['DeltaInMonths'][dl] == 13:\n",
+    "        if (absdeltas['HOST_SUBJECT_ID'][dl] not in host_13mo):\n",
+    "            host_13mo.update({absdeltas['HOST_SUBJECT_ID'][dl]:absdeltas['HOST_SUBJECT_ID'][dl]})\n",
+    "            uni_sub_del = uni_sub_del.append(absdeltas.loc[dl], ignore_index=True)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
```

Comment:
Can you provide a description of what the following cells are doing? It is not clear from looking at the code.